### ISSUE TITLE
Add browser side i18n example

### ISSUE THREAD

**Issue by ghost** [krakenjs/kraken-example-with-i18n#4]
I think that you should improve this example with a browser side rendering use case. 
Almost every app will need to implement it but the documentation (website,readme and examples) still doesn't address this issue.

**Comment by grawk** [comment]
Yep! you're absolutely right... We are in the process of updating our i18n functionality and examples. We need to be more transparent about our progress on this front. In the meantime though, it would be nice to update this example with a browser-based i18n template render. Marking as "help wanted" in case anybody else can get to it before the kraken team is able.

**Comment by killfish** [comment]
Can anyone point me in the right direction on how to implement browser side localized templates? Right now I'm compiling my browser side templates using webpack.

**Comment by aredridel** [comment]
That's actually a good strategy: the new makara components support loading a language package separately, so you can just compile a bundle and load templates along with the rest of your javascript.
